test/sym/assume.t

```diff
@@ -1,3 +1,8 @@
-assume extern module:
+symbolic extern module (assume and assert test):
   $ dune exec owi -- sym assume.wast
+  CHECK:
+  (i32.not (i32.to_bool symbol_0))
+  (i32.ge symbol_0 (i32 0))
+  (i32.to_bool symbol_0)
+  /CHECK OK
```

Comment:
We don't want this `CHECK ...` stuff, it should be commented in the `.ml` file (I believe it's in `choice_monad.ml` but I did not check.